packages/election/graphql/schema-types.js

```diff
@@ -0,0 +1,39 @@
+module.exports = `
+
+extend type User {
+  candidacies: [Candidacy!]!
+}
+
+type Election {
+  id: ID!
+  slug: String!
+  description: String
+  beginDate: DateTime!
+  endDate: DateTime!
```

Comment:
`beginDate` and `endDate` is unspecific. Is it the candidacy or the election phase?